open_alchemy/build/__init__.py

```diff
@@ -23,6 +32,29 @@
     _INIT_TEMPLATE = in_file.read()
 
 
+class PackageFormat(Flag):
+    """Define the available package formats for the build."""
+
+    NONE = auto()
+    SDIST = auto()
+    WHEEL = auto()
+
+
+def validate_dist_format(format_) -> None:
+    """
+    Ensure a valid package format is provided.
+
+    Raises a ``BuildError` exception if the format is invalid.
+
+    :param PackageFormat format_: the package format to validate
+    """
+    if format_ is not PackageFormat.NONE and PackageFormat.NONE in format_:
+        raise exceptions.BuildError(
+            "combining `PackageFormat.NONE` with another package type is "
+            "confusing, please be more specific"
```

Comment:
Might be nitpicking, it would be good to explain why. For example, that `PackageFormat.NONE` indicates that no additional package should be produced but another format does. It would also be good to show the format that was passed in the error to make it obvious.

Reply:
Addressed ✅